serenata_toolbox/serenata-toolbox

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import os
+
+from argparse import ArgumentParser
+
+from serenata_toolbox import log
+from serenata_toolbox.chamber_of_deputies.reimbursements import Reimbursements as ChamberDataset
+from serenata_toolbox.datasets import Datasets
+from serenata_toolbox.federal_senate.dataset import Dataset as SenateDataset
+
+
+def parser():
+    parser = ArgumentParser(description='Support Serenata de Amor and Rosie development.',
+                            usage='%(prog)s path [options]')
+    parser.add_argument('path', nargs='?', default='data/', help='directory for storing the datsets')
+    parser.add_argument('--dataset', '-d', nargs='+', help='the desired dataset to be downloaded')
+    parser.add_argument('--year', '-y', nargs='+', help='narrow the download to a list of years')
+    return parser.parse_args()
+
+
+def main():
+    args = parser()
+    path = args.path
+    if not os.path.isdir(path):
+        try:
+            os.mkdir(os.path.join(path))
+        except OSError:
+            log.error('Couldn\'t create the output directory')
+            raise
```

Comment:
As the `Dataset` class has a default directory, we could implement this `try`/`except` block in its `__init__` and make the CLI and the script behavior similar.

Reply:
Then you want me to remove this part here and add in `__init__` add there, having the check in both places?

Reply:
> having the check in both places?

No, we can just check if the directory exists only once, in `Dataset` ; )